src/content/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/migration-guide.mdx

```diff
@@ -0,0 +1,131 @@
+
+Node metrics are no longer scraped by default in the Prometheus agent, as it may cause metric duplication since it's normally used by New Relic's kubernetes integration.
+
+## Metric types [#metric-types]
+
+Prometheus OpenMetrics, `nri-prometheus`, as a custom solution, converts Prometheus metrics to New Relic metrics before sending them, by applying [mappings](/docs/infrastructure/prometheus-integrations/view-query-data/translate-promql-queries-nrql#compare) according to the metric metadata type reading from the scraped data: `# TYPE <metric_type>`.
+
+On the Prometheus agent, `newrelic-prometheus-agent`, which leverages a standard Prometheus server, metrics are sent directly to the New Relic's remote write endpoint, which handles this conversion based on metric names. Some metrics may not be properly converted by the remote write endpoint:
+
+- Prometheus Counter metrics **without** the name suffix `total`, `count`, `sum`, or `bucket` will be converted as **Gauge**.
+- Prometheus Gauge metrics **with** the name suffix `total`, `count`, `sum`, or `bucket` will be converted as **Counter**.
+
+To fix this, you can set up metric type mappings relabel options.
+
+## Transformations [#transformations]
+
+Prometheus OpenMetrics, `nri-prometheus`, transformations are now replaced with Prometheus relabel configurations. See a list of [examples of relabel configuration setups](/docs/infrastructure/prometheus-integrations/install-configure-prometheus-agent/setup-prometheus-agent#metric-label-transformations).
+
+## Self-instrumentation [#self-instrumentation]
+
+Similar to Prometheus OpenMetrics (`nri-prometheus`), the Prometheus agent (`newrelic-prometheus-agent`) self-scrapes internal metrics. These metrics have the `prometheus_` prefix and you can use them to observe the status of the Prometheus instance. By default, only a reduced set of these metrics are sent. You can find the list of metrics on the [default values.yaml file](https://github.com/newrelic/newrelic-prometheus-configurator/blob/main/charts/newrelic-prometheus-agent/values.yaml) of the chart, by searching for `job_name: self-metrics`.
+
+We also give you a curated dashboard for self-metrics with performance and health information, and also metrics volume sent. Install the quickstart containing the [dashboard for the Prometheus agent](https://newrelic.com/instant-observability/prometheus-agent) in your New Relic account.
+
+## Keep Prometheus OpenMetrics integration during the migration [#keep-openmetrics]
+
+Although our Prometheus OpenMetrics integration in Kubernetes has been replaced by the NR Prometheus Agent, it will be supported for a period of time. 
```

Comment:
which is this "period of time"? Can we state it clearly?

Reply:
Done (June 2023)